### DISCUSSION TITLE
Quantity of RAM

### DISCUSSION THREAD

**Discussion by Er9y714** [firemodels/fds#12217] [General]
I see a quantity of RAM as a device but I could not find further detail. Is that RAM usage of the complete simulation or the mesh the devices is located? Could you please give more insight? Thanks.

**Comment by mcgratta** [comment]
It is a feature that only works on linux computers. We use it for diagnostic purposes.

**Comment by Er9y714** [comment]
I am running simulations on Linux systems. I tried in one of the simulations and I measured a low RAM usage (~800 MB for 15 million cells). That is why I was curious if I interpret it correctly. Thanks.

**Comment by mcgratta** [comment]
It reports only the memory used for the process that controls the mesh in which the device is located. To get the total for the entire job, you would need to have a RAM device for every process.  If you do a "top" command, you will see that every process running FDS has its memory reported via RES (kB). That is essentially what RAM reports, but we convert to MB instead of kB.

**Comment by Er9y714** [comment]
Perfect. Thank you.
